cycode/cli/helpers/tf_content_generator.py

```diff
@@ -0,0 +1,29 @@
+import json
+from typing import List
+
+from cycode.cli.models import ChangeResource
+
+
+def generate_tf_content_from_tfplan(tfplan: str) -> str:
+    planned_resources = _extract_resources(tfplan)
+    return _generate_tf_content(planned_resources)
+
+
+def _extract_resources(tfplan: str) -> List[ChangeResource]:
+    tfplan_json = json.loads(tfplan)
+    resources: List[ChangeResource] = []
+    for change in tfplan_json.get('resource_changes', []):
+        resources.append(
+            ChangeResource(resource_type=change['type'], name=change['name'], values=change['change']['after'])
+        )
+    return resources
+
+
+def _generate_tf_content(resources: List[ChangeResource]) -> str:
+    tf_content = ''
+    for resource in resources:
+        tf_content += f'resource \"{resource.resource_type}\" \"{resource.name}\" {{\n'
+        for key, value in resource.values.items():
+            tf_content += f'  {key} = {json.dumps(value)}\n'
```

Comment:
`json.dumps(value)` works for you if its a dict? the inner fields format will be like json - `key: value` instead of `key = value"